public/animatedCounter.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8">
     <title>Animated Counter in JS</title>
-    <link rel="stylesheet" href="../assets/css/animatedCounter.css">
+    <link rel="stylesheet"  href="../assets/css/animatedCounter(updated).css">
```

Comment:
there is no need to create animatedCounter(updated).css
you can make the changes in the original css file itself
and remove the updated file